schema/src/ProjectInfo.yaml

```diff
@@ -41,12 +43,11 @@ classes:
         multivalued: true
         inlined: true
         inlined_as_list: true
-      location:
+      hasGeometry:
```

Comment:
can we change this to `hasLocation`? I know we talked about this but don't remember where we landed. To me it makes more sense to run sparql queries against location in natural language vs geometry which is more domain specific to the world of RDF.